src/App.tsx

```diff
@@ -2,13 +2,17 @@ import React from "react";
 import { AppRoutes } from "./routes";
 
 import { HTTP } from "./core/services/http";
+import Nav from './nav';
 
 HTTP.createClient("api");
 
 
 function App() {
     return (
-        <AppRoutes />
+        <div>
```

Comment:
change `<div>` to <>

i mean wrap the content in <> CODE </>, it is called React Fragment: 
https://stackoverflow.com/questions/47761894/why-are-fragments-in-react-16-better-than-container-divs